### PR TITLE
Abstract scrollbar visuals and behavior

### PR DESCRIPTION
**Pull request by jcmoore**
This work prepares for integration into mm-item-recycler

@dlasky 
@anthonykoerber

### CODE REVIEW

File: src/mm-scroll-panel/mm-scroll-panel.html [hunk 1]
```diff
@@ -16,13 +17,7 @@
...
 			</div>
 		</div>
-		<div id="scrollbarY"
-			 class="scrollbar-y" 
-			 style$="{{ _updateScrollbarStyle(scrollBarSize) }}"
-			 on-down="_onDown" 
-			 on-track="_onTrack">
-			<div class="bar"></div>
-		</div>
+		<mm-scrollbar-y id="scrollbar" interop="{{scrollInterop}}"></mm-scrollbar-y>
```
Comment (dlasky) [inline src/mm-scroll-panel/mm-scroll-panel.html]: interop

Comment (anthonykoerber) [inline src/mm-scroll-panel/mm-scroll-panel.html]: I'd have to agree here...

**Comment by anthonykoerber** [comment]
@jcmoore this looks/feels good - but it may be beneficial to get an IRL walk-thru, just for the sake of better understanding of the split into panel & bar.

File: src/mm-scroll-panel/mm-scroll-panel.html (hunk 1 above)
Comment (jcmoore) [inline src/mm-scroll-panel/mm-scroll-panel.html]: How about adapter?

**Comment by anthonykoerber** [comment]
+1

File: src/mm-scroll-panel/mm-scroll-panel.html [hunk 2]
```diff
@@ -7,6 +7,8 @@
 <link rel="import" href="../../bower_components/polymer/polymer.html">
 <link rel="import" href="../shared/behaviors/stylable.html">
 <link rel="import" href="../shared/behaviors/dommutable.html">
+<link rel="import" href="../shared/behaviors/mousewheelable.html">
```
Comment (dlasky) [inline src/mm-scroll-panel/mm-scroll-panel.html]: haha love it

**Pull request merged**